util.h

```diff
@@ -35,6 +35,7 @@ using namespace bricks::strings;
 
 namespace CTFO {
 static constexpr uint64_t ID_RANGE = static_cast<uint64_t>(1e18);
+static constexpr UID admin_uid = static_cast<UID>(1000000000000000001ull);
```

Comment:
Maybe `1 * ID_RANGE + 2` below? Or add a check in Auth handler. Now there is 1e-18 chance that someone will become an admin :)